src/ui/ExportToExcel/ExportToExcel.ts

```diff
@@ -12,28 +12,33 @@ export interface IExportToExcelOptions {
 }
 
 /**
- * This component allows users to export the current search results in a Microsoft Excel (.xlsx) format.
+ * This component allows the user to export their current search results to the Microsoft Excel format (.xlsx).
  * It populates the {@link Settings} component menu.
  */
 export class ExportToExcel extends Component {
   static ID = 'ExportToExcel';
+
   /**
-   * The options for the component
+   * The options for the ExportToExcel
```

Comment:
Just FYI, this comment is never shown, anywhere, in the generated documentation.

It is there only to ensure that the property of the `options` variable are documented.